[0m09:55:03.420019 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7f7b8c94ed70>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7f7b8a7ba440>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7f7b8a7bbdf0>]}


============================== 09:55:03.427047 | 18a4f55e-ead7-439a-8556-caae58e56d64 ==============================
[0m09:55:03.427047 [info ] [MainThread]: Running with dbt=1.8.4
[0m09:55:03.427346 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'log_cache_events': 'False', 'write_json': 'True', 'partial_parse': 'True', 'cache_selected_only': 'False', 'warn_error': 'None', 'fail_fast': 'False', 'version_check': 'True', 'log_path': 'logs', 'profiles_dir': '/home/akif/.dbt', 'debug': 'False', 'use_colors': 'True', 'use_experimental_parser': 'False', 'empty': 'None', 'quiet': 'False', 'no_print': 'None', 'log_format': 'default', 'introspect': 'True', 'invocation_command': 'dbt init agora_dbt', 'static_parser': 'True', 'target_path': 'None', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'send_anonymous_usage_stats': 'True'}
[0m09:55:03.443157 [debug] [MainThread]: Starter project path: /home/akif/.cache/pypoetry/virtualenvs/dbt-agora-L_hINThD-py3.10/lib/python3.10/site-packages/dbt/include/starter_project
[0m09:55:03.444058 [info ] [MainThread]: 
Your new dbt project "agora_dbt" was created!

For more information on how to configure the profiles.yml file,
please consult the dbt documentation here:

  https://docs.getdbt.com/docs/configure-your-profile

One more thing:

Need help? Don't hesitate to reach out to us via GitHub issues or on Slack:

  https://community.getdbt.com/

Happy modeling!

[0m09:55:03.444294 [info ] [MainThread]: Setting up your profile.
[0m09:56:06.169232 [info ] [MainThread]: Profile agora_dbt written to /home/akif/.dbt/profiles.yml using target's profile_template.yml and your supplied values. Run 'dbt debug' to validate the connection.
[0m09:56:06.171412 [debug] [MainThread]: Resource report: {"command_name": "init", "command_success": true, "command_wall_clock_time": 62.773724, "process_user_time": 1.047929, "process_kernel_time": 0.052126, "process_mem_max_rss": "92772", "process_out_blocks": "23416", "process_in_blocks": "0"}
[0m09:56:06.172919 [debug] [MainThread]: Command `dbt init` succeeded at 09:56:06.172689 after 62.78 seconds
[0m09:56:06.173895 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7f7b8c94ed70>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7f7b8ae86f50>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7f7b8a83d5d0>]}
[0m09:56:06.174831 [debug] [MainThread]: Flushing usage events
[0m13:18:29.310359 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7f28047adde0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7f2804b0fd90>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7f2804b0fd30>]}


============================== 13:18:29.311763 | 40fdc681-f00e-4480-b862-579ebc36748e ==============================
[0m13:18:29.311763 [info ] [MainThread]: Running with dbt=1.8.4
[0m13:18:29.312026 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'write_json': 'True', 'log_cache_events': 'False', 'partial_parse': 'True', 'cache_selected_only': 'False', 'profiles_dir': '/home/akif/.dbt', 'debug': 'False', 'fail_fast': 'False', 'log_path': 'logs', 'warn_error': 'None', 'version_check': 'True', 'use_colors': 'True', 'use_experimental_parser': 'False', 'empty': 'None', 'quiet': 'False', 'no_print': 'None', 'log_format': 'default', 'introspect': 'True', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'static_parser': 'True', 'target_path': 'None', 'invocation_command': 'dbt docs serve --port 8686', 'send_anonymous_usage_stats': 'True'}
[0m13:18:29.312264 [error] [MainThread]: Encountered an error:
Runtime Error
  No dbt_project.yml found at expected path /home/akif/Desktop/HCU/Agora/dbt_agora/dbt_project.yml
  Verify that each entry within packages.yml (and their transitive dependencies) contains a file named dbt_project.yml
  
[0m13:18:29.312944 [debug] [MainThread]: Resource report: {"command_name": "serve", "command_wall_clock_time": 0.025398994, "process_user_time": 0.513262, "process_kernel_time": 0.033113, "process_mem_max_rss": "84812", "process_in_blocks": "31728", "process_out_blocks": "16", "command_success": false}
[0m13:18:29.313248 [debug] [MainThread]: Command `dbt docs serve` failed at 13:18:29.313181 after 0.03 seconds
[0m13:18:29.313398 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7f28047adde0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7f2804b0fe20>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7f2804b0fd60>]}
[0m13:18:29.313562 [debug] [MainThread]: Flushing usage events
[0m13:33:58.641476 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7f97f4891e40>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7f97f4bffd90>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7f97f4bffd30>]}


============================== 13:33:58.642741 | 3892812f-468b-4f85-a117-74b913cda228 ==============================
[0m13:33:58.642741 [info ] [MainThread]: Running with dbt=1.8.4
[0m13:33:58.642967 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'write_json': 'True', 'log_cache_events': 'False', 'partial_parse': 'True', 'cache_selected_only': 'False', 'warn_error': 'None', 'debug': 'False', 'fail_fast': 'False', 'log_path': 'logs', 'version_check': 'True', 'profiles_dir': '/home/akif/.dbt', 'use_colors': 'True', 'use_experimental_parser': 'False', 'no_print': 'None', 'quiet': 'False', 'empty': 'False', 'log_format': 'default', 'static_parser': 'True', 'introspect': 'True', 'invocation_command': 'dbt run', 'target_path': 'None', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'send_anonymous_usage_stats': 'True'}
[0m13:33:58.643181 [error] [MainThread]: Encountered an error:
Runtime Error
  No dbt_project.yml found at expected path /home/akif/Desktop/HCU/Agora/dbt_agora/dbt_project.yml
  Verify that each entry within packages.yml (and their transitive dependencies) contains a file named dbt_project.yml
  
[0m13:33:58.643543 [debug] [MainThread]: Resource report: {"command_name": "run", "command_wall_clock_time": 0.024595987, "process_user_time": 0.482375, "process_kernel_time": 0.052257, "process_mem_max_rss": "84788", "process_out_blocks": "8", "command_success": false, "process_in_blocks": "0"}
[0m13:33:58.643776 [debug] [MainThread]: Command `dbt run` failed at 13:33:58.643735 after 0.02 seconds
[0m13:33:58.643929 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7f97f4891e40>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7f97f4bffe20>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7f97f4bffd60>]}
[0m13:33:58.644080 [debug] [MainThread]: Flushing usage events
